task.go

```diff
@@ -291,6 +311,40 @@ func (et *ExecutingTask) registerOutput(name string, o Output) {
 	et.outputs[name] = o
 }
 
+type ExecutionStats struct {
+	// Input nodes and their emitted points
+	// where input nodes - are batch/stream node
+	// for example: stream1 -> 12
+	Inputs map[string]int64
```

Comment:
We should update the struct to have both summary stats for the task and a map of all stats for the all nodes in the task.

maybe a structure like 
```
type ExecutionStats struct {
    TaskStats map[string]interface{} // stats name -> value i.e throughput -> 4.3 points /sec
    NodeStats map[string]map[string]interface{} // node name -> stat name -> value i.e influxdb_out -> point_written -> 1000
}
```

Then use the statsMap that each node has to populate the node stats maps.

Thoughts?